Automattic/jetpack
# tools: Add composer version check to composer-update-monorepo.sh

#### Changes proposed in this Pull Request:

Since this is the tool CI recommends when composer.lock needs updating,
we should try to inform the user when a wrong Composer version is likely
to lead to it still not passing the check.

#### Jetpack product discussion


p1634033604362000-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Install Composer 2.0.x or change the Composer version in `.github/versions.sh`. Run the script. See the warning.
